nixos/modules/misc/ids.nix

```diff
@@ -351,6 +351,7 @@ in
       hqplayer = 319;
       moonraker = 320;
       distcc = 321;
+      agate = 322;
```

Comment:
does this need a fixed uid across all nixos installations? user and group ids can be allocated dynamically by the systemd and remain consistent, cf https://github.com/NixOS/rfcs/pull/52

Reply:
ohh ok :+1:

Reply:
with `DynamicUser` set in the service this (and the matching group entry) shouldn't be necessary anymore

Reply:
Ah I thought I'd removed this. Done